### ISSUE TITLE
dogstatsd output does not work 

### ISSUE THREAD

**Issue by william-salt-cko** [falcosecurity/falcosidekick#225]
**Describe the bug**

When you try to configure dogstatsd forwarder, the the client errors on creation as it uses the normal statsd forwarder, which does not exist.
You can see this here
https://github.com/falcosecurity/falcosidekick/blob/1a7d7c790c9dc04f3efae9a2893fd129fb9c3519/outputs/statsd.go#L14
If you set both the dogstatsd forwarder and statsd forwarder in the config, falcosidekick panics.

**How to reproduce it**

Configure dogstatsd forwarder, or both statsd and dogstatsd forwarders

**Expected behaviour**

Dogstatsd forwarder works.

**Screenshots**

**Environment**

- Falco version: 2.21.0

**Comment by Issif** [comment]
Hello,

You're right, I noticed several issues with that output, I submitted a PR for fixing it: #227 

Do you want a docker image for testing the fix?

**Comment by willejs** [comment]
Thanks @Issif I was about to make a similar PR this morning! In the meantime, I have switched to stated to a datadog forwarder and made this PR to fix the chart too.
https://github.com/falcosecurity/charts/pull/212

Hopefully, we can get both merged 🙂

**Comment by Issif** [comment]
> Thanks @Issif I was about to make a similar PR this morning! In the meantime, I have switched to stated to a datadog forwarder and made this PR to fix the chart too.
> [falcosecurity/charts#212](https://github.com/falcosecurity/charts/pull/212)
> 
> Hopefully, we can get both merged 🙂

Thanks for the fix, and I like the tip with `checksum/config` for redeploying pods when `secrets.yaml` changes.

Another PR will have to be submitted for bumping up the dependency of `falco` chart https://github.com/falcosecurity/charts/blob/52e38480f85ee5cb8cb5048531c23db1db9b8007/falco/Chart.yaml#L22

**Comment by william-salt-cko** [comment]
Yeah, i was trying to figure out what was going on and kept having to kill pods 😆 
Do you think you could cut a release of this project? I think this is fixed now 🎉

**Comment by Issif** [comment]
> Yeah, i was trying to figure out what was going on and kept having to kill pods 😆
> Do you think you could cut a release of this project? I think this is fixed now 🎉

This is a bug fix, will do before end of the week.
